### DISCUSSION TITLE
Implementing OR Filtering

### DISCUSSION THREAD

**Discussion by mfinamore-sea** [apache/superset#23694] [Q&A / Help]
Hi everybody, I need to perform OR filtering on some datasets. I've tried to do it after the query to the backend but I didn't find a way to implement that. The idea is simple. For example i want to extract people who have "name" === "mike" or "age" >= "25". Can i do that during the query? Or should i manually filter the that after the query? Can you provide me some solutions?

Thanks for your help.

**Comment by paulobitar** [comment]
Maybe some jinja templates ?
